--- openssl.cnf.orig	2026-01-22 17:49:57.834423273 +0100
+++ openssl.cnf.new	2026-01-22 17:51:12.632183819 +0100
@@ -53,6 +53,14 @@
 [openssl_init]
 providers = provider_sect
 
+ssl_conf = ssl_module
+
+[ssl_module]
+system_default = tls_system_default
+
+[tls_system_default]
+Groups = x25519:prime256v1:x448:secp521r1:secp384r1:ffdhe2048:ffdhe3072:brainpoolP256r1
+
 # List of providers to load
 [provider_sect]
 default = default_sect
